src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -47,10 +47,17 @@ public class DrivebaseSubsystem extends SubsystemBase {
 		Rotation2d.fromDegrees(-332.841)
 	};
 
-	public static final double MAX_DRIVE_SPEED_METERS_PER_SEC = 4.4196;
-	public static final Rotation2d MAX_ROTATION_SPEED_PER_SEC = Rotation2d.fromRotations(2.2378);
-
-	private static final double WHEEL_DIAMETER_METERS = 0.0889;
+	// max drive speed is from SDS website and not calculated with robot weight
+	public static final double MAX_DRIVE_SPEED_METERS_PER_SEC = IS_COMP ? 4.4196 : 4.1148;
+	// see how this is calculated here: https://www.desmos.com/calculator/rrglfgzqnh
+	// it's basically just rotations_per_sec = velocity/(2*pi*turning radius)
+	public static final Rotation2d MAX_ROTATIONS_PER_SEC =
+			Rotation2d.fromRotations(IS_COMP ? 1.171 : 2.303);
```

Comment:
This seems off, as the competition bot isn't twice as wide as bonk. I calculate the comp bot as 32.29744572" (0.820355121 meters) diagonally and bonk as 24.04163056: (0.610657416 meters) diagonally.

Reply:
recalculated rotation speed in 6afa911